.circleci/config.yml

```diff
@@ -47,10 +47,11 @@ jobs:
             curl -L https://github.com/commercialhaskell/stack/releases/download/v2.3.1/stack-2.3.1-linux-x86_64.tar.gz | tar zx -C /tmp
             sudo mv /tmp/stack-2.3.1-linux-x86_64/stack /usr/bin
             sudo apt-get update
-            # ncat utility from nmap needed to test socket connection with curl <  7.40
-            sudo apt-get install -y libgmp-dev postgresql-client nmap
+            sudo apt-get install -y libgmp-dev postgresql-client
             sudo apt-get install -y --only-upgrade binutils
             stack setup
+            stack install hpc-lcov
```

Comment:
where and how exactly do I add this to any of the cabal or stack files instead of calling this manually here? would be important to be reflected in the cache key through that as well.

Reply:
Hmm.. maybe it could go in `stack.yaml` -> `extra-deps`.

Then below use like `stack exec hpc-lcov && coveralls < lcov.info`?

(Not sure if it'd work)

Reply:
That sounds like it makes sense. But how do I add it there? there are some hashes in there - I am not supposed to put those in by hand, am I? I tried `stack install` locally, but that didn't add anything. Any stack command that adds those as extra-deps?

Reply:
IIRC, the hashes are only needed when stack doesn't have the package on its repos.

Looks like hpc-lcov is there though: https://www.stackage.org/lts-15.8/hoogle?q=hpc-lcov

You could try it like:
```
extra-deps:
- hpc-lcov-1.0.1
```

Actually, I've just noticed the hash is here: https://www.stackage.org/lts-16.19/package/hpc-lcov-1.0.1

```
This version can be pinned in stack with:hpc-lcov-1.0.1@sha256:f7831e13530634f0c2f4f2e613ddc15541f78ca45e4214e73f23efcfb90f68cb,2845
```